manifests/s/sigoden/Dufs/0.31.0/sigoden.Dufs.installer.yaml

```diff
@@ -0,0 +1,27 @@
+# Created with YamlCreate.ps1 v2.2.1 $debug=NVS1.CRLF.7-3-1.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: sigoden.Dufs
+PackageVersion: 0.31.0
+InstallerType: zip
+ReleaseDate: 2022-11-12
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Microsoft.VCRedist.2015+.x64
+Installers:
```

Comment:
```suggestion
  - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

Reply:
Apparently, I added a bad line ending or other invisible character with my previous suggestion. I believe that I have corrected it in this suggestion. Thank you for your patience.

Reply:
![image](https://user-images.githubusercontent.com/56779163/218659411-e82a4c0a-8dcb-49c6-9209-b5e355e903fe.png)
And nothing wrong here. 🤔